app/plugins/music_service/mpd/index.js

```diff
@@ -955,18 +957,14 @@ ControllerMpd.prototype.createMPDFile = function (callback) {
 
       var mixerdev = '';
       var mixerstrings = '';
-      if (outdev != 'softvolume') {
-        var realDev = outdev;
-        if (outdev.indexOf(',') >= 0) {
-          mixerdev = 'hw:' + outdev;
-          outdev = 'hw:' + outdev;
+      if (!softvolume) {
+        if (realDev.indexOf(',') >= 0) {
+          mixerdev = 'hw:' + realDev;
         } else {
-          mixerdev = 'hw:' + outdev;
-          outdev = 'hw:' + outdev + ',0';
+          mixerdev = 'hw:' + realDev;
```

Comment:
Not sure if this is an oversight, but isn't this the `else` branch here the same as in the `if` branch?